unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/TaskbarHUD/TaskbarHUDController.cs

```diff
@@ -369,7 +369,10 @@ public void OnCloseWindowToggleInputPress()
         if (mouseCatcher.isLocked)
             return;
 
-        view.chatButton.SetToggleState(true);
+        // this gives focus to chat input field and breaks the InputProcessor.Modifier.FocusNotInInput functionality
+        // when pressing escape and not having cursor locked
+        // like moving the character around without cursor locking
+        //view.chatButton.SetToggleState(true);
```

Comment:
I left the comment just as a warning in case it is required to activate the chat button for some reason in the future. I didnt see any troubles with the tests i made regarding to this issue, so it would be cleaner to remove the code